playback.json

```diff
@@ -157,6 +157,24 @@
         }
       ]
     },
+    {
+      "name": "Hide Waiting For Game",
+      "authors": [
+        "Nikki",
+        "UnclePunch"
+      ],
+      "description": [
+        "Hides the waiting for game text and slippi.gg text and disables the SFX played before a replay starts"
```

Comment:
Perhaps a disclaimer here might be nice: "With this enabled, there will be no indication between matches that Melee is still running"